### PR TITLE
Add tests to validators

### PR DESCRIPTION
**Pull request by JjuliaP**
Create tests to list_of_int_validator
Create tests to string_validator
Create tests to json_loads

## JIRA

* [Main JIRA ticket]()

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/tests/unittests/validator/testvalidators.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import datetime
```
Comment (redmyt) [inline eventually/tests/unittests/validator/testvalidators.py]: maybe it is better to rename dir from `validator` to `validators`?

Comment (JjuliaP) [reply eventually/tests/unittests/validator/testvalidators.py]: dir? ok

File: eventually/tests/unittests/validators/testvalidators.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+from utils.utils import *
+
+
+class ValidatorsTestCase(TestCase):
+    """TestCase for validators"""
+
+    def test_list_of_int_validator_list_int(self):
+        """Method that tests list_of_int_validator method with list of int enter parameter"""
+        my_list_first = list_of_int_validator([1, 2, 3])
+        self.assertEqual(my_list_first, True)
```
Comment (lhalam) [inline eventually/tests/unittests/validators/testvalidators.py]: use assertTrue

File: eventually/tests/unittests/validators/testvalidators.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+    def test_list_of_int_validator_tuple_int(self):
+        """Method that tests list_of_int_validator method with tuple of int enter parameter"""
+        my_list = list_of_int_validator((1, 2, 3))
+        self.assertEqual(my_list, True)
+
+    def test_list_of_int_validator_int(self):
+        """Method that tests list_of_int_validator method with int enter parameter"""
+        my_list = list_of_int_validator(1)
+        self.assertEqual(my_list, False)
```
Comment (lhalam) [inline eventually/tests/unittests/validators/testvalidators.py]: use assertFalse

File: eventually/tests/unittests/validators/testvalidators.py (hunk 2 above)
Comment (JjuliaP) [reply eventually/tests/unittests/validators/testvalidators.py]: Done

File: eventually/tests/unittests/validators/testvalidators.py (hunk 3 above)
Comment (JjuliaP) [reply eventually/tests/unittests/validators/testvalidators.py]: Done

**Review by lhalam — APPROVED** [review]

**Pull request merged**